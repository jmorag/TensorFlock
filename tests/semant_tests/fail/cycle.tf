main = f;

f : Nat;
f = x;

x : Nat;
x = f;
