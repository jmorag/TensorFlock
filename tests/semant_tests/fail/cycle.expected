Fatal error: exception Failure("Cycle in variable declaration")
